### ISSUE TITLE
Support Conan

### ISSUE THREAD

**Issue by tequdev** [Xahau/xahaud#402]
Currently I'm using the conan-strict branch by @dangell7 as a base during development.
It would be great if this could be used in the dev branch as well.

https://github.com/Xahau/xahaud/tree/conan-strict

However, Boost 1.86 used in the current dev branch causes build errors, so I have downgraded to 1.83 in my environment.
https://github.com/tequdev/xahaud/commit/ec9c62541ecf1a559bb96b5e12f30c6a0c7c2671

**Comment by dangell7** [comment]
We can do it the trick is the release builder. So you would need to keep the build files in another directory and copy them.

**Comment by dangell7** [comment]
and I think make changes to the [CMakeLists.txt](https://github.com/Xahau/xahaud/blob/dev/CMakeLists.txt)

**Comment by tequdev** [comment]
The current release-builder has the path hard-coded as /data/builds, which is not available on macos where these directories are not accessible even with admin.

**Comment by dangell7** [comment]
Yeah it needs to be run on a ubuntu machine. You could add a mac os if/else or just change it for development

**Comment by RichardAH** [comment]
As long as the release builder is adjusted to continue working I'm ok with the "primary" build method becoming conan
